package/xochitl/package

```diff
@@ -5,12 +5,12 @@
 pkgnames=(xochitl)
 pkgdesc="Read documents and take notes"
 url=https://remarkable.com
-pkgver=0.0.0-11
+pkgver=0.0.0-12
 timestamp=2021-11-29T22:06Z
 section="readers"
 maintainer="Mattéo Delabre <spam@delab.re>"
 license=MIT
-installdepends=(findutils)
+installdepends=(findutils toltec-meta)
```

Comment:
I think it would make more sense to add this transitional dependency to `toltec-bootstrap`, as the `xochitl` package was not previously installed by default (only as a dependency of some launchers).

Reply:
While that is true, there have been several users who don't have toltec-bootstrap installed as they installed toltec before it was its own package. This would get more coverage. How about both?

Reply:
That sounds good! I added a note above the dependency in both recipes to give context for future readers, and to remind ourselves to remove those dependencies after a while.